iwyu.cc

```diff
@@ -2611,6 +2541,14 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
       // in source files, or for builtins, or for friend declarations.
       if (!IsInHeader(decl) || IsFriendDecl(decl))
         return pair(false, nullptr);
+      for (auto paramIt = decl->param_begin(); paramIt != decl->param_end();
```

Comment:
`FunctionDecl` has a nice `ArrayRef<ParmVarDecl*> parameters()` wrapper to make this a little neater;
```
for (const ParmVarDecl* param : decl->parameters()) {
   // param instead of *paramIt
}
```